numpy/ma/core.py

```diff
@@ -4332,13 +4310,12 @@ def __long__(self):
             raise MaskError('Cannot convert masked element to a Python long.')
         return long(self.item())
 
-
-    def get_imag(self):
+    @property
+    def imag(self):
         """
-        Return the imaginary part of the masked array.
+        Get and set the imaginary part of the masked array.
```

Comment:
I would drop these first 3 words. I'd expect property docstrings to be describing a thing, not an action.

Reply:
Looking again, this has no setter anyway.